### ISSUE TITLE
Bulk delete

### ISSUE THREAD

**Issue by iammattmartin** [jstanden/cerb#844]
In the latest 9.1 release, bulk delete _tickets_ or attachments does not appear to work. 

It lets you submit the task but the worklist just refreshes without any action taken.

If you delete the _tickets_ individually on the worklist, this works.

**Comment by jstanden** [comment]
We'll take a look. Thanks!

**Comment by jstanden** [comment]
I was able to bulk delete attachments from a worklist in my tests.

Were you using an admin-level worker?

As for messages, do you mean tickets? I'm not sure we've ever had bulk update on message worklists.

**Comment by iammattmartin** [comment]
Sorry, replace messages with tickets.

https://scrn.at/yUc03QVp.mov is what happens, as an admin.

**Comment by jstanden** [comment]
Hi Matt,

Your video clearly shows an issue, but I'm not able to reproduce it.

Can you share the output of this MySQL/MariaDB query after you try a bulk update?

~~~
select * from context_bulk_update where created_at > unix_timestamp() - 300 \G
~~~

Are there any SQL errors in your PHP error log? It may be another MariaDB issue.

**Comment by iammattmartin** [comment]
It looks like it was MySQL related but due to the last upgrade that went wrong.

Also, I think may be unrelated, we're getting PHP "errors" of logged REPLACE INTO commands. It seems from the last upgrade too. 

```
Jan  8 12:38:25 hd ool www: [0]  ::SQL:: REPLACE INTO worker_view_model (worker_id,view_id,is_ephemeral,class_name,title,options_json,columns_json,columns_hidden_json,params_query,params_editable_json,params_required_json,params_required_query,params_default_json,render_page,render_total,render_limit,render_sort_json,render_subtotals,render_template) VALUES (1,'api_search_cerberusweb_contexts_ticket',1,'View_Ticket','Search Results','[]','[\"t_last_wrote_address_id\",\"t_updated_date\",\"t_group_id\",\"t_bucket_id\",\"t_owner_id\"]','[]','','{\"cd0f189b2f5c7535\":{\"field\":\"*_status\",\"operator\":\"in\",\"value\":[\"deleted\"]},\"b2561f39362c5853\":{\"field\":\"cf_26\",\"operator\":\"like\",\"value\":\"disabled*\"}}','[]','','{\"*_status\":{\"field\":\"*_status\",\"operator\":\"in\",\"value\":[\"open\",\"waiting\"]},\"*_groups_of_worker\":{\"field\":\"*_groups_of_worker\",\"operator\":\"=\",\"value\":1}}',0,1,1,'{\"t_updated_date\":false}','','contextlinks_chooser')
```

Could it be due something left debugging in code?

**Comment by jstanden** [comment]
If you re-run one of those `REPLACE INTO worker_view_model` queries from the log, what error does MySQL return?

**Comment by iammattmartin** [comment]
That appears unrelated hence why I deleted it. It seems there is an issue with the database behind haproxy. Can't see any other apps have the issue but managed to find a PHP error in /var/www/libs/devblocks/api/services/bootstrap/database.php on line 305 which seems to indicate MySQL disconnected but still a bit of a mystery.

The original error isn't that but appears to be something that didn't run correctly on upgrade which I manually fixed and now seems to work just ok :)
